kayenta-web/config/kayenta.yml

```diff
@@ -50,6 +50,16 @@ kayenta:
 #          - METRICS_STORE
 #        endpoint.baseUrl: https://app.datadoghq.com
 
+  newrelic:
+    enabled: false
+#    accounts:
+#    - name: my-newrelic-account
+#      apiKey: xxxx
+#      applicationKey: xxxx
+#      supportedTypes:
+#      - METRICS_STORE
+#      endpoint.baseUrl: https://insights-api.newrelic.com
```

Comment:
If this endpoint is static and the same for all New Relic customers, then you might consider making it a constant rather then a configuration property.